pvlib/tracking.py

```diff
@@ -21,7 +21,7 @@ class SingleAxisTracker(PVSystem):
 
     axis_azimuth : float, default 0
```

Comment:
editorial: I'd list `axis_aximuth` first, since `axis_tilt` references it.

As long as we're working at the docstring, the phrase "(i.e., the y-axis defined by axis_azimuth)" doesn't help `axis_tilt` since we haven't described the coordinate system here.

Reply:
I believe this could theoretically break some folks code if they're using positional arguments if we want the docstring order to match the argument order.

Reply:
Also the [`singleaxis()` documentation](https://pvlib-python.readthedocs.io/en/latest/generated/pvlib.tracking.singleaxis.html) is more detailed. Perhaps we should add a note in `SingleAxisTracker` to direct them to read the `singleaxis` docstring, rather than repeating it again?

Reply:
Withdraw my suggestion to re-order arguments.